pkg/env/prod.go

```diff
@@ -117,6 +120,12 @@ func newProd(ctx context.Context, log *logrus.Entry, instancemetadata instanceme
 	p.clustersGenevaLoggingPrivateKey = clustersGenevaLoggingPrivateKey
 	p.clustersGenevaLoggingCertificate = clustersGenevaLoggingCertificates[0]
 
+	e2eStorageAccountKey, err := p.GetSecret(ctx, E2EStorageAccountKeySecretName)
```

Comment:
so for now im fetching the storage key from svc KV we may want to avoid that and fetching from one and unique KV for all the regions no ?

Reply:
let me know if this is better that way ?